### ISSUE TITLE
Old non-existent analysers showing in Cortex after an upgrade

### ISSUE THREAD

**Issue by daskydasky** [TheHive-Project/Cortex#234]
### Request Type
Bug

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | Debian
| OS version (client)       | Windows 10
| Cortex version / git hash   | 3.0.0-1
| Package Type              | deb
| Browser type & version    | Chrome 77.0

### Problem Description
Old non-existent analysers showing in Cortex after an upgrade. 
I have upgraded TheHive / Cortex / Cortex Analysers and in Hive could not see any analysers. 
Basically in Cortex on analysers tab still showing previously installed analysers.
Analysers catalogue is up to date. 

### Possible Solutions
As a workaround, I have created a new organization.

**Comment by RtKelleher** [comment]
I opened a similar ticket and it was closed, you can remove the old analyzers in Elasticsearch by deleting the original index entry. I'm unsure why they arn't removed by the software, but that was the oldway I could get rid of the old analyzers.
